app/src/main/res/layout-land/fragment_edit_category_options.xml

```diff
@@ -70,6 +70,42 @@
         app:layout_constraintEnd_toStartOf="@+id/remove_category_button"
         app:layout_constraintTop_toTopOf="@+id/edit_options_back_button" />
 
+    <TextView
+        android:id="@+id/empty_phrases_text"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/custom_category_empty"
+        android:textColor="@color/textColor"
+        android:textStyle="bold"
+        android:textSize="22sp"
```

Comment:
We're trying to avoid hardcoded values in our layout files. Could you move these to the corresponding `dimens` files? There should be one for each form factor. It might also be worth making a style if these all share similar properties.